README.md

```diff
@@ -25,7 +25,6 @@ The Home page is now available at "http://localhost:81/".
 You need also to create your own GitHub [personal token](https://github.com/settings/tokens) and export it:
 
 ```bash
-export SYMFONY_PHPUNIT_VERSION=5.7
```

Comment:
unit test won't work with PHPUnit < 5

Reply:
Now it will :) I fixed it

Reply:
you're using PHPUnit 6 but the phpunit bridge will install PHPUNit 4.8 by default xD

The test suite was already compatible with PHPUnit 6 ;)

I still love you ahaha

Reply:
Test wasn't working for me with PHPUnit 6, and PHPUnit bridge was installing the latest PHPUnit  for me -_-'